runtime/lua/vim/diagnostic.lua

```diff
@@ -1440,9 +1440,10 @@ M.handlers.signs = {
 
     local numhl = opts.signs.numhl or {}
     local linehl = opts.signs.linehl or {}
+    local line_count = api.nvim_buf_line_count(bufnr)
```

Comment:
This will fail if the buffer isn't loaded.

Reply:
Ah, right, thanks!

Reply:
You can probably move the `api.nvim_buf_is_loaded(bufnr)` check higher up in this function.

Reply:
Alright I've moved it out up top, also changing the parameter from `diagnostic.bufnr` to just the given `bufnr`. Hopefully this is fine